web/api/tests/web_api.c

```diff
@@ -18,72 +54,124 @@ RRDHOST *__wrap_rrdhost_find_by_hostname(const char *hostname, uint32_t hash)
 void __wrap_finished_web_request_statistics(
     uint64_t dt, uint64_t bytes_received, uint64_t bytes_sent, uint64_t content_size, uint64_t compressed_content_size)
 {
+    (void)dt;
+    (void)bytes_received;
+    (void)bytes_sent;
+    (void)content_size;
+    (void)compressed_content_size;
 }
 
 char *__wrap_config_get(struct config *root, const char *section, const char *name, const char *default_value)
 {
     if (!strcmp(section, CONFIG_SECTION_WEB) && !strcmp(name, "web files owner"))
         return "netdata";
+    (void)root;
+    (void)default_value;
     return "UNKNOWN FIX ME";
 }
 
 int __wrap_web_client_api_request_v1(RRDHOST *host, struct web_client *w, char *url)
 {
-    printf("api requests: %s\n", url);
+    char url_repr[160];
+    repr(url_repr, sizeof(url_repr), url, strlen(url));
+    printf("web_client_api_request_v1(url=\"%s\")\n", url_repr);
+    check_expected_ptr(host);
+    check_expected_ptr(w);
+    check_expected_ptr(url_repr);
```

Comment:
```
[  ERROR   ] --- No entries for symbol url_repr.
web/api/tests/web_api.c:80: error: Could not get value to check parameter url_repr of function __wrap_web_client_api_request_v1
web/api/tests/web_api.c:288: note: Previously declared parameter value was declared here

```
You should use `check_expected_ptr(url)`.

Reply:
I've fixed it the other way around, by changing the `expect_string` to use url_repr. The reasoning here is that it makes it easier to check for nulls and high-ascii characters in the interface while it is still changing. Later on it might more sense to flip both to `url`.

Reply:
You can't check `expect_string` ahainst `url_repr` because `__wrap_web_client_api_request_v1` doesn't have such parameter.